code/modules/photography/camera/camera.dm

```diff
@@ -15,26 +15,35 @@
 	flags_1 = CONDUCT_1
 	slot_flags = ITEM_SLOT_NECK
 	materials = list(MAT_METAL = 50, MAT_GLASS = 150)
-	var/flash_enabled = TRUE
+	var/obj/item/disk/holodisk/disk
+	var/pictures_left
```

Comment:
Cleaner this way if a map wants to have a camera with 1 picture but you can still refill to 10 or whatever
```suggestion
	var/pictures_left = 0
```